scripts/docker/zeppelin-server/Dockerfile

```diff
@@ -41,43 +41,43 @@ ENV LANG=en_US.UTF-8 \
     LC_ALL=en_US.UTF-8 \
     JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64 \
     VERSION="${version}" \
-    Z_HOME="/opt/zeppelin" \
+    ZEPPELIN_HOME="/opt/zeppelin" \
     ZEPPELIN_ADDR="0.0.0.0" \
```

Comment:
@Leemoonsoo 
I think you should also set `HOME` in this place to fix your problem with Helium in this Dockerfile as well.

Reply:
right. I updated this, too.